### PR TITLE
Ensure clicking "Log" in command menu works for workflows in GScan sidebar

### LINKED ISSUE #1507: GScan workflow command menu - "Log" command doesn't work when not already viewing that workflow
### Reproducible Examples

1. Open the dashboard view
2. Click on a workflow icon in the GScan sidebar to open the command menu
3. Click "Log"
4. Nothing happens

Another:

1. Open a workflow
2. Click on a _different_ workflow icon in the GScan sidebar to open the command menu
3. Click "Log"
4. The log view for _this_ workflow opens

### Expected Behaviour

It should navigate to the workflow whose icon you clicked and open the log view for it

### PR DESCRIPTION
**Pull request by MetRonnie**
Closes #1507

**Check List**

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Tests are included 
- [x] `CHANGES.md` entry included if this is a change that can affect users

### CODE REVIEW

File: src/components/cylc/cylcObject/Menu.vue [hunk 1]
```diff
@@ -77,7 +77,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               <v-btn
                 icon
                 variant="text"
-                :disabled="isEditable(authorised, mutation)"
+                :disabled="!isEditable(mutation, authorised)"
```
Comment (MetRonnie) [inline src/components/cylc/cylcObject/Menu.vue:80]: N.B. before this, `isEditable` returned the opposite of what its name suggests

**Review by wxtim — APPROVED** [review]
- [x] Read the code: Looks good.
- [x] Tried it with both scenarios given in #1507 : Happy that you now get a log, and it's for the correct workflow. 😄

File: src/components/cylc/cylcObject/Menu.vue [hunk 2]
```diff
@@ -344,14 +321,34 @@ export default {
...
-        this.$workflowService.apolloClient
-      )
-      this.showMenu = false
+
+      if (mutation.name === 'log') {
+        this.$router.push({
+          name: 'workspace',
+          params: {
+            workflowName: this.node.tokens.workflow
+          }
```
Comment (oliver-sanders) [inline src/components/cylc/cylcObject/Menu.vue:333]: An explanatory comment linking back to this PR would be good (context easily forgotten).

**Review by oliver-sanders — APPROVED** [review]

**Comment by MetRonnie** [comment]
Merging with 2 approvals, [latest change pushed](https://github.com/cylc/cylc-ui/compare/256413e26a82f5ea0b728dc8c8c4e61e2c884ac9..2e3f2bff0326be9c1e9a68c2189d03ef1018d69d) to address review comments should be uncontroversial

**Pull request merged**